src/liballoc/heap.rs

```diff
@@ -26,6 +26,10 @@ pub unsafe fn allocate(size: usize, align: usize) -> *mut u8 {
 ///
 /// On failure, return a null pointer and leave the original allocation intact.
 ///
+/// If the allocation was relocated, the memory at the passed-in pointer is undefined after the
+/// call.
+///
```

Comment:
Our vim files auto-wrap at 100, so doing this is a manual thing, which kinda sucks. I'll still do it here, this just keeps cropping up.